src/include/miopen/generic_search.hpp

```diff
@@ -469,38 +358,29 @@ auto GenericSearch(const Solver s,
     HeartBeat<PerformanceConfig> heartbeat;
     heartbeat.Start();
 
-    profile_h.EnableProfiling(true);
     for(const auto& current_config : all_configs)
     {
         float elapsed_time = 0.0f;
         int ret            = 0;
         MIOPEN_LOG_I2('#' << n_current << '/' << n_failed << '/' << n_runs_total << ' '
                           << current_config);
 
-        const auto current_solution = s.GetSolution(context, current_config, true);
-        if((tweak == SearchTweak::WorkspaceInsteadOfXBuffer ||
-            tweak == SearchTweak::WorkspaceInsteadOfWeightsBuffer) &&
-           default_solution.workspce_sz != current_solution.workspce_sz)
+        ConvSolution current_solution;
+        Invoker invoker;
+
+        try
```

Comment:
How do we check that WS size does not depend on PerformanceConfig?

Reply:
fixed that

Reply:
Thanks!